osbenchmark/utils/git.py

```diff
@@ -32,15 +33,16 @@
 def probed(f):
     def probe(src, *args, **kwargs):
         # Probe for -C
-        if not process.exit_status_as_bool(lambda: process.run_subprocess_with_logging(
-                "git -C {} --version".format(io.escape_path(src)), level=logging.DEBUG), quiet=True):
-            version = process.run_subprocess_with_output("git --version")
-            if version:
-                version = str(version).strip()
-            else:
-                version = "Unknown"
-            raise exceptions.SystemSetupError("Your git version is [%s] but Benchmark requires at least git 1.9. "
-            "Please update git." % version)
+        try:
+            out, err, status = process.run_subprocess_with_out_and_err(
+                "git -C {} --version".format(io.escape_path(src)))
+        except FileNotFoundError:
+            status = 1
+        if status != 0:
+            raise exceptions.SystemSetupError("Error invoking 'git', please install (or re-install).")
+        match = re.search(r'.* ([0-9]+)\.([0-9]+)\..*', out)
+        if not match or int(match.group(1)) < 2:
```

Comment:
Instead of having `2` hardcoded here and in the exception, it'd be better to include it as a global variable, such as `REQUIRED_GIT_MAJOR_VER` at the top of this file and referenced in the if statement and exception. 
```
if not match or int(match.group(1)) < REQUIRED_GIT_MAJOR_VER:
    raise exceptions.SystemSetupError(f"OpenSearch Benchmark requires at least version {REQUIRED_GIT_MAJOR_VER} of git.  You have {out}.  Please update git.")
```

Reply:
Good point.  The earlier version was hardcoded at 1.9, but this is cleaner.